plugins/GSdx/GSSettingsDlg.cpp

```diff
@@ -372,6 +378,28 @@ void GSSettingsDlg::UpdateControls()
 		bool ocl = renderer == GSRendererType::DX9_OpenCL || renderer == GSRendererType::DX1011_OpenCL || renderer == GSRendererType::OGL_OpenCL;
 		bool null = renderer == GSRendererType::Null;
 
+		int selected_CRC = SendMessage(GetDlgItem(m_hWnd, IDC_CRC_LEVEL), CB_GETCURSEL, 0, 0);
+
+		if (prev_renderer == GSRendererType::Undefined)
+			prev_renderer = static_cast<GSRendererType>(i);
+		else if (renderer == GSRendererType::OGL_HW && prev_renderer != GSRendererType::OGL_HW) {
+			if (selected_CRC == 3 && non_default_CRC != 3) {
```

Comment:
Would making `non_default_CRC` a bool work? (I don't know whether it could be made to work, the current logic does make me think quite hard though.)

Side note: Just my personal opinion - if an `if` or `else` condition uses `{}`, then I think it looks nicer if the matching `else`/`if` also uses `{}`.